### ISSUE TITLE
Cannot Access Postal Web UI

### ISSUE THREAD

**Issue by absolutezack** [postalserver/postal#2509]
## Describe the bug

I just do new installation postal on my new production server. I already follow the instruction. But I get an error when I accessed web postal UI, I got this error message on my browser "Access to 192.168.1.135 was deniedYou don't have authorization to view this page. HTTP ERROR 403". 

## To Reproduce

Port 5000 is already listen to my IP address (192.168.1.135)

I also got this error message on postal logs:
web_1       | [8] [2023-06-21T03:18:03.185] ERROR -- : [ActionDispatch::HostAuthorization::DefaultResponseApp] Blocked host: 192.168.1.135
web_1       | 192.168.1.241 - - [21/Jun/2023:03:18:03 +0000] "GET / HTTP/1.1" 403 - 0.0038

## Environment details 

 - Ubuntu 20.04
 - Mozila, chrome, Microsoft edge 

## Additional information/context
I also do an "postal upgrade" on my postal lab server, I got no error on the upgrading process. But I faced the same issue on my lab server, I cannot access web postal UI

**Comment by sinnbeck** [comment]
Is the server on the same local network as your computer? Generally you want it to be accessible on an external ip and access it on that

What did you use as webserver?

**Comment by absolutezack** [comment]
Yes it is on the same network on my computer, I also can lookup port 5000 from my computer to postal server.

**Comment by sinnbeck** [comment]
Give it an external ip and a domain and use caddy or nginx (or similar) to set up a proper webserver as explained in the docs.

https://docs.postalserver.io/install/installation#caddy
